src/actor/Claimer.spec.ts

```diff
@@ -395,5 +449,86 @@ describe('Claimer', () => {
         )
       )
     })
+    it('Should throw, when claimer tries to make a request for attestation, without a PE-enabled identity', async () => {
+      blockchainApi.query.attestation.attestations.mockReturnValue(
+        mockChainQueryReturn('attestation', 'attestations', [
+          cType.hash,
+          attester.getAddress(),
+          undefined,
+          0,
+        ])
+      )
+
+      const claimerWithoutPE = await Identity.buildFromURI('//bob', {
+        peEnabled: false,
+      })
+
+      const { message: initAttestation } = await Attester.initiateAttestation(
+        attester,
+        claimerWithoutPE.getPublicIdentity()
+      )
+
+      await expect(
+        Claimer.requestAttestation(
+          claim,
+          claimerWithoutPE,
+          attester.getPublicIdentity(),
+          {
+            initiateAttestationMsg: initAttestation,
+          }
+        )
+      ).rejects.toThrowError(ERROR_IDENTITY_NOT_PE_ENABLED())
+    })
+
+    it('Should throw, when claimer tries to make a credential, without a PE-enabled identity', async () => {
+      blockchainApi.query.attestation.attestations.mockReturnValue(
+        mockChainQueryReturn('attestation', 'attestations', [
+          cType.hash,
+          attester.getAddress(),
+          undefined,
+          0,
+        ])
+      )
+
+      const claimerWithoutPE = await Identity.buildFromURI('//bob', {
+        peEnabled: false,
+      })
+
+      const {
+        message: initAttestation,
+        session: attersterSession,
+      } = await Attester.initiateAttestation(
+        attester,
+        claimer.getPublicIdentity()
+      )
+
+      const {
+        message: requestAttestation,
+        session: claimerSession,
+      } = await Claimer.requestAttestation(
+        claim,
+        claimer,
+        attester.getPublicIdentity(),
+        {
+          initiateAttestationMsg: initAttestation,
+        }
+      )
+
+      const { message: attestationMessage } = await Attester.issueAttestation(
+        attester,
+        requestAttestation,
+        claimer.getPublicIdentity(),
+        attersterSession,
+        true
+      )
+
+      await expect(
+        Claimer.buildCredential(
+          claimerWithoutPE,
+          attestationMessage,
+          claimerSession
+        )
+      ).rejects.toThrowError(ERROR_IDENTITY_NOT_PE_ENABLED())
+    })
```

Comment:
I would also add an expected success when using the claimer with PE

```suggestion
      await expect(
        Claimer.buildCredential(claimer, attestationMessage, claimerSession)
      ).resolves.toBeTruthy()
    })
```

Reply:
fixed in ea38166a4ce1b9c442db0264098e7f7ff080847f